lumicks/pylake/correlated_stack.py

```diff
@@ -129,36 +223,45 @@ def from_data(cls, data, name=None, start_idx=0, stop_idx=None):
         new_correlated_stack.stop_idx = (new_correlated_stack.src.num_frames if stop_idx is None else stop_idx)
         return new_correlated_stack
 
-    def plot(self, frame=0, **kwargs):
+    def plot(self, frame=0, channel="rgb", show_title=True, **kwargs):
         """Plot image from image stack
 
         Parameters
         ----------
         frame : int, optional
             Index of the frame to plot.
+        channel : 'rgb', 'red', 'green', 'blue', None; optional
+            Channel to plot for RGB images (None defaults to 'rgb')
+            Not used for grayscale images
+        show_title : bool, optional
+            Controls display of auto-generated plot title
         **kwargs
             Forwarded to :func:`matplotlib.pyplot.imshow`.
         """
         import matplotlib.pyplot as plt
 
         default_kwargs = dict(
-            cmap='gray'
+            cmap='gray',
+            vmax=None
         )
+        kwargs = {**default_kwargs, **kwargs}
 
-        image = self._get_frame(frame).data
-        plt.imshow(image, **{**default_kwargs, **kwargs})
+        image = self._get_frame(frame)._get_plot_data(channel, vmax=kwargs["vmax"])
+        plt.imshow(image, **kwargs)
 
-        if self.num_frames == 1:
-            plt.title(self.name)
-        else:
-            plt.title(f"{self.name} [frame {frame}/{self.num_frames}]")
+        if show_title:
+            if self.num_frames == 1:
+                plt.title(self.name)
+            else:
+                # display with 1-based index for frames and total frames
+                plt.title(f"{self.name} [frame {frame+1}/{self.num_frames}]")
 
     def _get_frame(self, frame=0):
         if frame >= self.num_frames or frame < 0:
             raise IndexError("Frame index out of range")
         return self.src.get_frame(self.start_idx + frame)
 
-    def plot_correlated(self, channel_slice, frame=0, reduce=np.mean):
+    def plot_correlated(self, channel_slice, frame=0, channel="rgb", vmax="auto", reduce=np.mean):
```

Comment:
Would it make sense to make this more uniform with `plot` (e.g. forwarding `vmax` to `imshow`). Would mean removing `auto` as option as well. Was the `auto` option tested? Because it doesn't seem to be supported by `_get_plot_data` anymore?

Reply:
Missed this one. I completely took out the `vmax` kwarg for this method since other optional plotting parameters aren't supported at this point (`cmap`, etc..) So for `channel="rgb"` the data will be normalized and all other options the data will be returned as is.

Reply:
Yeah, it happens. I guess I forgot to add a test for the actual plots at the time. Could add them in a separate test which just spins up the plots to prevent a regression in the future, That or maybe make a separate ticket for it so we don't forget.

We could consider adding the other `kwargs` that `plot` has to this one and handle it similarly, forwarding `kwargs` to `imshow`. Then `vmax` could also be used with `plot_correlated`. It'd be pretty nice to have, since you said by default they can be very dim. But if you don't want the scope creep, that's also fine, we can make a separate ticket out of that. Maybe that's better, since this one has been in flight for a while now :)

Just one general comment, since we don't know whether people are using named arguments, adding an argument in the middle could cause issues for some people. We should either put new arguments at the end or mention very clearly in the changelog that an argument was added in the middle.

Reply:
Yes, it'd be nice to have the optional plotting arguments forwarded to `imshow` but I figured save that for a separate ticket since it's not really in the scope of this one (and there's already some things out of scope potentially that have made their way into this PR)

And I see what you mean about the argument order, I'll move `channel` to the end